pages/Prefixes,-Suffixes-&-Meta.md

```diff
@@ -33,9 +33,9 @@ Prefixes and suffixes are split up into two parts
 * **Weight** - this is just a number which determines the priority of the prefix/suffix. A higher number = a higher weight and a higher priority. 
 * **Value** - what the actual value of the prefix is.
 
-A prefix of "[Admin] ", set with a weight of 100 translates into the permission: `"prefix.100.[Admin] "`.
+A prefix of `"[Admin] "`, set with a weight of 100 translates into the permission: `prefix.100."[Admin] "`.
```

Comment:
I actually think this wording is even more confusing..

To set the prefix `[Admin] ` (with a trailing space)

The command is: `/lp group admin meta addprefix 100 "[Admin] "` - and the permission in the web-editor would look like `"prefix.100.[Admin] "`

There shouldn't be a quote in the middle of the node

Reply:
Interesting, last time I needed quotes for prefix with spaces, probably changed. But the original issue is the `"` makes it looks like its suppose to be included in the node itself.

Reply:
Yeah, I think so

Reply:
Alright done!